### DISCUSSION TITLE
Adding additional fields to one Sunbird RC instance vs Multi instance of RC

### DISCUSSION THREAD

**Discussion by shravya-techjini** [Sunbird-RC/community#259] [General]
Dear Team, 

This query is related to design activities of remote Volunteering building block which eVidyaloka is planning to contribute.
As part of design activity, We wanted to add few more registry like Volunteer registry  etc and add additional fields to the User and existing entities in Sunbird RC. 

We were exploring two options:

a) Use Sunbird RC, Create new registry and add additional fields to existing entities. There will be only 1 RC instances. 
b) Enhance Sunbird RC, add new registry and add additional fields to the existing entities. In this, we will deploying an instance of Sunbird RC , add the configuration. 

We have following queries on the options
a)Do we have an API which will allow to add additional fields to User and other existing entities of RC
b)Your thoughts on which option is suitable in our context 

Regards,
Shravya

**Comment by parthlawate** [comment]
Hi @shravya-techjini .. Please take a look at the documentation here. This should help : https://docs.sunbirdrc.dev/use/schema-configuration

Sunbird RC is a configurable system which will let you define your own entities and fields.. While there are reference solutions the core building block can be used to create any kind of registries.
